cmd/plugins/juju-metadata/signmetadata.go

```diff
@@ -82,6 +82,9 @@ func process(dir, key, passphrase string) error {
 	logger.Debugf("processing directory %q", dir)
 	// Do any json files in dir
 	filenames, err := filepath.Glob(filepath.Join(dir, "*"+simplestreams.UnsignedSuffix))
+	if err != nil {
+		return err
```

Comment:
This wouldn't have been a bug, but it's safer now. filepath.Glob only returns errors when the pattern is invalid, so I guess this would never happen anyway.